ADR/0041-integration-service-upstreaming.md

```diff
@@ -0,0 +1,95 @@
+# XX. Creating Artifacts from Snapshots for testing and release
```

Comment:
Can we change this to ADR 43? I am trying to deconflict the in-review ADRs.

Reply:
Ack, updated